04/app.js

```diff
@@ -6,4 +6,36 @@ const user = {
         month: '04',
         year: '1985'
     }
-}
\ No newline at end of file
+}
+
+console.log(
+    user.born.day,
+    user.born.month,
+    user.born.year,
+)
+
+function checkBirthsDate() {
```

Comment:
Nie chodzi o porównanie 2 dat urodzin, tylko czy ktoś obchodzi urodziny w tym dniu czyli:
1) należy porównać dzień i miesiąc
2) należy pobrać aktualny dzień oraz miesiąc, aby móc to sprawdzić (trzeba wykorzystać `new Date()` oraz `getDate()` i `getMonth()`)

Reply:
Zadanie zostało zmienione.